### PR TITLE
Fsm-v1.3

### PR DESCRIPTION
**Pull request by AmanKumar-eGov**

### CODE REVIEW

File: core-services/README.md [hunk 1]
```diff
@@ -5,3 +5,11 @@ DIGIT Core Platform consists of the foundational microservices for authenticatio
 #### DIGIT is released under MIT License
 
 
+<<<<<<< HEAD
+License
+DIGIT is released under MIT License
+
```
Comment (talele08) [inline core-services/README.md:11]: Why are this conflict not resolved?

File: CODEOWNERS [hunk 2]
```diff
@@ -7,10 +15,15 @@
 * frontend/ @sathishp-eGov @talele08 @kavi-egov @sirishrao-egov
 * edcr/ @pradeep
 * finance/ @elzan
+>>>>>>> master
```
Comment (talele08) [inline CODEOWNERS:18]: Resolve the conflict everywhere in the PR

File: municipal-services/property-services/src/main/java/org/egov/pt/producer/Producer.java [hunk 3]
```diff
@@ -1,20 +0,0 @@
-package org.egov.pt.producer;
```
Comment (talele08) [inline municipal-services/property-services/src/main/java/org/egov/pt/producer/Producer.java:1]: Why is this file deleted? @hinamakhija-eGov please confirm if it is correct

**Review by talele08 — CHANGES_REQUESTED** [review]
I have done only high level review of the overall structure. @AmanKumar-eGov @jagankumar-egov please approve sanitation and UI changes

File: municipal-services/property-services/src/main/java/org/egov/pt/producer/Producer.java (hunk 3 above)
Comment (hinamakhija-eGov) [reply municipal-services/property-services/src/main/java/org/egov/pt/producer/Producer.java:1]: Since enc-client had the same name of Producer file, this file had to be renamed for proper functioning and it was renamed as PropertyProducer. 
Ref: https://github.com/egovernments/DIGIT-Dev/blob/develop/municipal-services/property-services/src/main/java/org/egov/pt/producer/PropertyProducer.java

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/fsm/src/components/MobileInbox.js [hunk 4]
```diff
@@ -82,10 +82,18 @@ const MobileInbox = ({
         <div className="filters-container">
           {/* {!isFstpOperator && !isSearch && <ApplicationLinks linkPrefix={parentRoute} isMobile={true} />} */}
           <div style={{ display: "none" }}>
+<<<<<<< HEAD
```
Comment (jagankumar-egov) [inline frontend/micro-ui/web/micro-ui-internals/packages/modules/fsm/src/components/MobileInbox.js:85]: @AmanKumar-eGov merge conflicts were not resolved

**Review by jagankumar-egov — CHANGES_REQUESTED** [review]

**Comment by jagankumar-egov** [comment]
@AmanKumar-eGov & @talele08 
 can you check this pr 
https://github.com/egovernments/DIGIT-OSS/pull/71
